backend/api/models/FuelCodeStatus.py

```diff
@@ -0,0 +1,56 @@
+"""
+    REST API Documentation for the NRS TFRS Credit Trading Application
+
+    The Transportation Fuels Reporting System is being designed to streamline
+    compliance reporting for transportation fuel suppliers in accordance with
+    the Renewable & Low Carbon Fuel Requirements Regulation.
+
+    OpenAPI spec version: v1
+
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+from django.db import models
+
+from api.managers.FuelCodeStatusManager import FuelCodeStatusManager
+from api.models.mixins.DisplayOrder import DisplayOrder
+from auditable.models import Auditable
+
+
+class FuelCodeStatus(Auditable, DisplayOrder):
```

Comment:
Will this table have columns for description, display order, effective date, expiration date, create/update user and date? Wondering if these are created in a different part of the code?

Reply:
I left out the description, effective dates and expiration dates from this table as they look unnecessary here. But they should be easy to add if it's our standard.

Create/Update user and date should be inherited from Auditable.
Same goes with Display Order

Reply:
Ok - I was thinking because this table would be a code/lookup table we'd want it to be consistent with the other code/lookup tables in the model. I agree that description could be redundant because the code values are in essence the descriptions of the values... but we should consider keeping the effective/expiration as this supports showing the code value being "valid/active/etc").  And we'd expire the code rather than delete it. And because the create/update columns are on all tables, they should be here too.

That said, other options for showing "validity" of a code value, could be handled using an active_ind flag instead. It's different than the other code tables, but certainly something we can chat about.